README.md

```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for homework 01
+## Heba's homework 
+*I hope this line to appear italic*  
+
+I have two difficulties:
+1. The professor is proceeding very fast :) 
+1. The book is hard to follow 
```

Comment:
I am surprised by your list here. Making a list, you used "1." in the syntax for both points, but the list was formatted correctly. I did not know you could do that. Thank you for teaching me something about Markdown!